yash-datalake/nested/datalakeAnalytics.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Data Lake Store account to create."
+      }
+    },
+    "adlAnalyticsName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Data Lake Analytics account to create."
+      }
+    },
+    "adlStoreName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Data Lake Store account to create."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.DataLakeStore/accounts",
+      "name": "[parameters('adlStoreName')]",
+      "apiVersion": "2016-11-01",
+      "location": "East US 2"

```

Comment:
this cannot be hardcoded

Reply:
Sure, we will make the required changes...

Reply:
Replaced the hardcoded value.